src/test/main.test.js

```diff
@@ -39,8 +38,16 @@ const logDetailed = true;
 
 
 // these mirror the keys in scripts/run-swarms.rb
-const master_pub_key = "MFYwEAYHKoZIzj0CAQYFK4EEAAoDQgAEE/Yeq9sYdyeou+TnNEJjMnuntrzqcFIfIHd49LW461d55TY4hVX66ZXXGvAWRqMVMeELtYuKGYU44bPaxTb1ig==";
-const master_priv_key = "MHQCAQEEIEOd7E9zSxgJjtpGzK/gHl0vVSOZ2iF3TY50InD67BnHoAcGBSuBBAAKoUQDQgAEE/Yeq9sYdyeou+TnNEJjMnuntrzqcFIfIHd49LW461d55TY4hVX66ZXXGvAWRqMVMeELtYuKGYU44bPaxTb1ig==";
+const master_pub_key = "-----BEGIN PUBLIC KEY-----\n\
```

Comment:
This is a change in contract to our users. Do we gain any benefit from requiring the header/footer? If not, I'd rather keep the user expectation the same.

Reply:
I think I'd prefer the full header/footer model so that we don't make any transformations of the key. It was like that with the old crypto setup because I had to manually interpret the base64 string. The new js crypto setup (and openssl) use the full key, so I don't see any reason to deviate from that structure.

https://bluzelle.slack.com/archives/G9LG77PEV/p1564511398032000